src/components/Stepper/src/Stepper.vue

```diff
@@ -95,6 +103,12 @@ export default {
 		minVal() {
 			return Number.parseInt(this.min, 10);
 		},
+		resolvedColor() {
+			return this.color || this.theme.colors.contrast['color-100'] || '#cccccc';
+		},
+		resolvedTextColor() {
+			return this.textColor || this.theme.colors.contrast['color-800'] || '#000000';
+		},
```

Comment:
if you'd like to follow the convention established in some of the other components, in order to resolve a potential theme value you would use the `resolveThemeableProps` utility function, button example: https://github.com/square/maker/blob/beta/src/components/Button/src/Button.vue#L231

And you would add a `stepper` property to the default theme which would look something like this:
```js
// in default theme object
stepper: {
  color: '@colors.contrast.color-100',
  textColor: '@colors.contrast.color-800',
}
```
button example: https://github.com/square/maker/blob/beta/src/components/Theme/src/default-theme.js#L13-L21

and then add in the defaults for those colors in the theme itself as well:
```js
// in default theme object
colors: {
  color-100: '#cccccc',
  color-800: '#000000',
}
```